### PR TITLE
Create Storybook stories for Penumbra colors

### PR DESCRIPTION
**Pull request by jessepinho**
Creates [this page](https://penumbra-ui-preview--pr1500-jessepinho-colors-xw8bzgdf.web.app/?path=/docs/ui-library-colors--docs) in Storybook

### CODE REVIEW

File: packages/ui/src/Colors.stories.tsx
Comment (VanishMax) [inline packages/ui/src/Colors.stories.tsx:3]: question: are these changes coming from #1489? It confused me since Technical and other components were not reviewed yet but the questions are mostly about them, not the color stories 😅

It's just that even if we merge this PR now, these changes will need to be reviewed once again with other changes

Comment (jessepinho) [reply packages/ui/src/Colors.stories.tsx:3]: @VanishMax ah, yes, but I haven't made that PR ready for review yet (oops). My plan was to wait until they're all approved, then merge them into each other and finally into `main`. If you give this the 👍🏻 , I'll wait to merge it until #1489 is approved and merged.

Comment (jessepinho) [reply packages/ui/src/Colors.stories.tsx:3]: Just marked #1489 ready for review as well

**Review by TalDerei — APPROVED** [review]

**Pull request merged**
